src/SFA.DAS.RoatpGateway.Web.UnitTests/Services/OrganisationChecks/Orchestrator/OrganisationStatusTests.cs

```diff
@@ -110,7 +117,11 @@ public void check_organisation_status_orchestrator_builds_with_company_details_o
                 ProviderName = ProviderName,
                 ProviderStatus = ProviderStatus,
                 VerifiedByCompaniesHouse = true,
-                VerifiedbyCharityCommission = false
+                VerifiedbyCharityCommission = false,
+                VerificationDetails = new List<VerificationDetails>
+                {
+                    new VerificationDetails {VerificationAuthority="Charity Commission", VerificationId=charityNumber}
```

Comment:
What is this new verification details about? Was this missing before?

Reply:
I need to extract the charity number from somewhere as we're now using the charityNumber in the call to the new endpoint (rather than the applicationId to the old endpoint), and you can extract it from the VerificationDetails returned from ukrlp, and this sets that up for the test to succeed.